rules/browser.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
```

Comment:
We are in es5 server side

Reply:
You are able to override your own configuration:
```
// http://eslint.org/docs/user-guide/configuring
module.exports = {
  extends: 'prestashop',
  env: {
    es6: false,
    es5: true,
  }
};
```

Reply:
It also working with rules if you project have a rule you really don't want.